source/reference/mongosync/mongosync-behavior.txt

```diff
@@ -233,6 +233,46 @@ number of writes may differ between source and destination.
 Transactions might not appear atomically on the destination cluster.
 Retryable writes may not be retryable on the destination cluster.
 
+.. _c2c-verifier-disclaimer:
+
+Embedded Verifier Disclaimer
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Starting in 1.9, ``mongosync`` includes an embedded verifier to
+perform a series of verification checks on all supported
+collections on the destination cluster to confirm that
+it was successful in transferring documents from the
+source cluster to the destination.
+
+When you start the ``mongosync`` process, it provides a
+disclaimer advising the user that the verifier is enabled
+by default.
+
+.. code-block:: none
+
+   Embedded verification is enabled by default with compatible beta features.
+   Oplog Rollover Resilience and migrations with source clusters running MongoDB 5.0+
```

Comment:
Can we use the disclaimer from the `prod` [binary](https://github.com/10gen/mongosync/blob/0c56c40977cf4500118c1741aed646527f2b5ddb/internal/mongosync/app_prod.go#L14) instead of the `beta` one? They are almost the same except for the reference to beta features.

Reply:
Fixed